src/Tizen.NUI/src/public/Visuals/TextVisual.cs

```diff
@@ -23,10 +23,11 @@ namespace Tizen.NUI
     /// <since_tizen> 3 </since_tizen>
     public class TextVisual : VisualMap
     {
+        static private float defaultPointSize = 12;
```

Comment:
is this similar with DALi?

Reply:
I changed value 12 to 18, which is used in DALi as a default value for TextLabel